rotkehlchen/assets/resolver.py

```diff
@@ -105,15 +121,31 @@ def resolve_asset_to_class(
         - WrongAssetType: if the asset is resolved but the class is not the expected one.
         - UnknownAsset: if the asset was not found in the database.
         """
+        from rotkehlchen.constants.assets import CONSTANT_ASSETS  # pylint: disable=import-outside-toplevel  # isort:skip  # noqa: E501
+        from rotkehlchen.globaldb.handler import GlobalDBHandler   # pylint: disable=import-outside-toplevel  # isort:skip  # noqa: E501
+
         resolved_asset = AssetResolver().resolve_asset(
             identifier=identifier,
             form_with_incomplete_data=form_with_incomplete_data,
         )
-        if isinstance(resolved_asset, expected_type) is False:
-            raise WrongAssetType(
+        if isinstance(resolved_asset, expected_type) is True:
+            # resolve_asset returns Asset, but we already narrow type with the if check above
+            return resolved_asset  # type: ignore
+
+        if identifier in CONSTANT_ASSETS:
+            log.debug(f'Attempt to get asset_type for asset with {identifier} using the packaged database')  # noqa: E501
+            resolved_asset = GlobalDBHandler().resolve_asset(
```

Comment:
This logic is also not needed. The assets is returned by the `resolve_asset` method either from the user globald or the packaged global db